example/signature-verifier.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * This utility module helps to demonstrate following features
+ * a. Signing a message by an Ethereum user
+ * b. Finding the account address using which the message was signed
+ */
+var Web3 = require('../index.js');
+var ethURL = ""; 
+var defaultAc = ""; 
+var defaultAcPWD=""; 
+var signatureContractCodeReadable="\n\tcontract SignatureVerifier {\n\t\tfunction verify( bytes32 hash, uint8 v, bytes32 r, bytes32 s) \n"+ 
+    "\t\tconstant returns(address returnAddress) {\n \t\t\treturnAddress = ecrecover(hash, v, r, s);\n\t\t}\n\t}\n\n";
+
+var sigContractInstance = null;
+var sigContractAddress= ""; 
+var sigContractInstance = null;
+var strAbi='[{"constant":true,"inputs":[{"name":"hash","type":"bytes32"},{"name":"v","type":"uint8"},{"name":"r","type":"bytes32"},{"name":"s","type":"bytes32"}],"name":"verify","outputs":[{"name":"returnAddress","type":"address"}],"payable":false,"type":"function"}]';
+var signMessage=""; 
+
+var ethWeb3 = null;
+
+function setContractAddress(conAddress){
+    sigContractAddress = conAddress;
+}
+
+function setAccount(act){
+    defaultAc = act;
+}
+
+function setPassword(pwd){
+    defaultAcPWD = pwd;
+}
+
+function setEthereumURL(url){
+    ethURL = url;
+}
+
+function setMessage(msg){
+    signMessage = msg;
+}
+
+function initializeEthereumConnection(){
+   if(ethWeb3!=null && ethWeb3.isConnected()==true)  {
+    return true;
+  }
+  
+  ethWeb3 = new Web3(new Web3.providers.HttpProvider(ethURL));
+  
+  if(ethWeb3.isConnected()==true){
+      if(defaultAc==''){
+        defaultAc=ethWeb3.eth.accounts[1];
```

Comment:
Oh seems quite a change in features which i need to follow :) 

With existing release came across many queries in this regard mostly people not able to verify who signed after signing a message. So prepared this example.

Good to see with newer version these examples are clearly mentioned in the docs.

By the way when are you expecting for the final release of version 1?